sphinx/source/quickstart/build.rst

```diff
@@ -50,6 +50,7 @@ The full list of build switches can be obtained by running:
 * **SAN**: Boolean. Build unit tests with Address and Undefined behaviour sanitizers enabled. Default to OFF.
 * **TARGET**: String, one of ``sgx``, ``virtual``, or ``sgx;virtual``. Defaults to ``sgx;virtual``, which builds both "virtual" enclaves and actual SGX enclaves.
 * **VERBOSE_LOGGING**: Boolean. Enable all logging levels. Default to OFF.
+* **PBFT**: Boolean. Enables PBFT as the consensus protocol. Default to OFF.
```

Comment:
Can you please change if OFF raft will be used.

Also outside of this PR we should consider changing the PBFT flag to be a consensus flag which has string values of raft and pbft where the default is raft?

Reply:
Yes, we should have -DCONSENSUS, like we have -DTARGET. Eventually I think we'd prefer a runtime flag, because 1. the CI and release will be much lighter, and 2. users might conceivably want to switch an app without a code change. It could be a single-shot parameter passed to the first node/network (effectively part of genesis), and joiners could discover it on join. On the other hand, it will make for somewhat bigger binaries.